api/decision.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	DecisionPath = "/decisions"
+	DecisionPath = "/decisions/generic"
```

Comment:
We need to set up a redirect from `/decisions/*` (excluding "/decisions/generic(|/*)") to `/decisions/generic/$1` for backwards compatibility!

Reply:
I've handled this by simply relying on the middleware order. 
This does present an issue for the metrics, and I'm not sure how to handle that.